mtgjson5/providers/tcgplayer.py

```diff
@@ -213,6 +215,42 @@ def generate_today_price_dict(
 
         return dict(combined_listings)
 
+    def generate_mtgjson_sealed_product_objects(
+        self, group_id: Optional[Any]
+    ) -> List[MtgjsonSealedProductObject]:
+        """
+        Builds MTGJSON Sealed Product Objects from TCGPlayer data
+        :param group_id: group id for the set to get data for
+        :return: A list of MtgjsonSealedProductObject for a given set
+        """
+        if not self.__keys_found:
+            LOGGER.warning("Keys not found for TCGPlayer, skipping")
+            return []
+
+        sealed_data = get_tcgplayer_sealed_data(group_id)
+
+        mtgjson_sealed_products = []
+
+        for product in sealed_data:
+            sealed_product = MtgjsonSealedProductObject()
+            sealed_product.name = product["cleanName"]
+            sealed_product.identifiers.tcgplayer_product_id = str(product["productId"])
+            sealed_product.uuid = str(
```

Comment:
Would prefer this to be added to the add_uuid function already defined in set_builder. Maybe move that function to utils and change it. I just want all UUID stuff in one location